docs/guidance/build/change_history.md

```diff
@@ -1,3 +1,6 @@
+```{workedexample} Change history
+:tags: release,record
+```
```

Comment:
I don't know if it's required, but to avoid potential future issues, please add a newline after the closing backticks in all files.